Rules/Strings.resx

```diff
@@ -726,4 +726,13 @@
   <data name="ProvideDefaultParameterValueName" xml:space="preserve">
     <value>ProvideDefaultParameterValue</value>
   </data>
+  <data name="AvoidUsingDeprecatedManifestFieldsCommonName" xml:space="preserve">
+    <value>Avoid Using Deprecated Manifest Fields</value>
+  </data>
+  <data name="AvoidUsingDeprecatedManifestFieldsDescription" xml:space="preserve">
+    <value>Some manifest fields are obsolte in the latest PowerShell version. Please update with the latest manifest fields in modules to avoid PowerShell version inconsistency.</value>
```

Comment:
Typo in "obsolete"

Also, can we point to the incorrect manifest fields in the description?